### PR TITLE
AP-4706: remove task list classes

### PR DESCRIPTION
**Pull request by agoldstone93**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-4706)

Remove custom css where possible and replace with standard govuk classes
Rename from `app-task-list` to `sort-transactions`

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- The standards in the [Git Workflow document on Confluence](https://dsdmoj.atlassian.net/wiki/spaces/ATPPB/pages/4602855954/Git+Workflow) should be followed
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by colinbruce — APPROVED** [review]
LGTM :+1:

File: app/views/providers/income_summary/_income_type_item.html.erb [hunk 1]
```diff
@@ -8,29 +8,24 @@
...
-      <div class="govuk-hint" style="margin-bottom: 0px">
-        <%= t("transaction_types.hints.#{name}") %>
-      </div>
-    <% end %>
     <% if error_message %>
       <p id='<%= "error-#{name}" %>' class="govuk-error-message govuk-!-padding-top-2"><%= error_message %></p>
     <% end %>
   </h2>
-  <%= content_tag(:div, class: "app-task-list__items", id: "list-item-#{name}") do %>
+  <%= content_tag(:div, class: "govuk-!-padding-bottom-9 govuk-!-padding-left-0", id: "list-item-#{name}") do %>
```
Comment (naseberry) [inline app/views/providers/income_summary/_income_type_item.html.erb:23]: nit: can we remove `content_tag` and have just a `<div>` instead?
It seems redundant here to ruse the rails tag helper when we can write the html tag.

**Review by naseberry — APPROVED** [review]
aside from my comment, everything else looks ok

File: app/views/providers/outgoings_summary/_outgoing_type_item.html.erb [hunk 2]
```diff
@@ -8,24 +8,19 @@
...
-      <div class="govuk-hint" style="margin-bottom: 0px">
-        <%= t("transaction_types.hints.#{name}") %>
-      </div>
-    <% end %>
     <% if error_message %>
       <p id='<%= "error-#{name}" %>' class="govuk-error-message govuk-!-padding-top-2"><%= error_message %></p>
     <% end %>
   </h2>
-  <%= content_tag(:div, class: "app-task-list__items", id: "list-item-#{name}") do %>
+  <%= content_tag(:div, class: "sort-transactions__items", id: "list-item-#{name}") do %>
```
Comment (naseberry) [inline app/views/providers/outgoings_summary/_outgoing_type_item.html.erb:18]: nit: can we remove `content_tag` and have just a `<div>` instead?

Comment (agoldstone93) [reply app/views/providers/outgoings_summary/_outgoing_type_item.html.erb:18]: Yep, sorry, should've checked for other instances. Will fix now

File: config/locales/en/providers.yml [hunk 3]
```diff
@@ -820,21 +820,15 @@ en:
       show:
         error: Select yes if they have received other benefits
     income_summary:
-      income_type_item:
```
Comment (colinbruce) [inline config/locales/en/providers.yml:825]: Are these handled elsewhere now?  
I always get the fear when locale stuff is deleted, not moved! :)

**Review by colinbruce — APPROVED** [review]
One non-blocking observation, but otherwise additional changes look good :+1:

File: config/locales/en/providers.yml (hunk 3 above)
Comment (agoldstone93) [reply config/locales/en/providers.yml:825]: Sorry, was probably a bit heavy handed of me. I couldn't see where they were being used, but I can add them back in to be on the safe side

**Review by naseberry — APPROVED** [review]
nice improvements to the partials

File: config/locales/en/providers.yml (hunk 3 above)
Comment (colinbruce) [reply config/locales/en/providers.yml:825]: We do run an i18n test for expected keys being missing and if that didn't trigger it's probably okay :+1:

**Pull request merged**